libs/solana/src/lib/helpers/generate-close-account-transaction.ts

```diff
@@ -0,0 +1,46 @@
+import { TransactionType } from '@kin-tools/kin-memo'
+import { generateKinMemoInstruction } from '@kin-tools/kin-transaction'
+import { createCloseAccountInstruction, getAssociatedTokenAddress, TOKEN_PROGRAM_ID } from '@solana/spl-token'
+import { Transaction, TransactionInstruction } from '@solana/web3.js'
+import { GenerateCloseAccountTransactionOptions } from '../interfaces'
+import { getPublicKey } from './get-public-key'
+
+export async function generateCloseAccountTransaction({
+  appIndex,
+  lastValidBlockHeight,
+  latestBlockhash,
+  mintFeePayer,
+  mintPublicKey,
+  signer,
+}: GenerateCloseAccountTransactionOptions): Promise<Transaction> {
+  // Create objects from Response
+  const mintKey = getPublicKey(mintPublicKey)
+  const feePayerKey = getPublicKey(mintFeePayer)
+
+  // Create Memo Instruction for KRE Ingestion - Must be Memo Program v1, not v2
+  const appIndexMemoInstruction = generateKinMemoInstruction({
+    appIndex,
+    type: TransactionType.None,
+  })
+
+  // Get AssociatedTokenAccount
+  const associatedTokenAccount = await getAssociatedTokenAddress(mintKey, signer.publicKey)
+
+  // Create Transaction
+  const instructions: TransactionInstruction[] = [
+    appIndexMemoInstruction,
+    createCloseAccountInstruction(TOKEN_PROGRAM_ID, associatedTokenAccount, feePayerKey, []),
```

Comment:
Please check the properties of this `createCloseAccountInstruction` 

<img width="646" alt="image" src="https://user-images.githubusercontent.com/36491/177765121-c38dec9b-4996-489b-a6e4-586c3c8df994.png">


We only need to pass in `account`, `destination` and `authority`.